docs/modules/ROOT/pages/includes/quarkus-vault.adoc

```diff
@@ -192,7 +192,8 @@ Possible values are:
 * low: display all secrets.
 * medium: display only usernames and lease ids (ie: passwords and tokens are masked).
 * high: hide lease ids and dynamic credentials username.
---|`low`, `medium`, `high` 
+-- a|
```

Comment:
For my own understanding, why `a`?

Reply:
The formatter that runs during the build makes this change. I have no idea why.